### PR TITLE
[MM-60827] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/suggestion/suggestion_list.tsx' with FormattedMessage 

### PR DESCRIPTION
**Pull request by Camillarhi**
#### Summary

Replace FormattedMarkdownMessage in `webapp/channels/src/components/suggestion/suggestion_list.tsx` with FormattedMessage 
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28499
Jira https://mattermost.atlassian.net/browse/MM-60827
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/suggestion/suggestion_list.tsx [hunk 1]
```diff
@@ -225,7 +224,7 @@ export default class SuggestionList extends React.PureComponent<Props> {
                 className='suggestion-list__no-results'
                 ref={this.contentRef}
             >
-                <FormattedMarkdownMessage
+                <FormattedMessage
                     id='suggestion_list.no_matches'
                     defaultMessage='No items match __{value}__'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/suggestion/suggestion_list.tsx:229]: `__` is for bold as well

File: webapp/channels/src/components/suggestion/suggestion_list.tsx [hunk 2]
```diff
@@ -225,11 +224,12 @@ export default class SuggestionList extends React.PureComponent<Props> {
...
             >
-                <FormattedMarkdownMessage
-                    id='suggestion_list.no_matches'
-                    defaultMessage='No items match __{value}__'
+                <FormattedMessage
+                    id='suggestionList.noMatches'
+                    defaultMessage='No items match <b>{value}</b>'
                     values={{
                         value: this.props.pretext || '""',
+                        b: (chunks: string) => <b>{chunks}</b>
```
Comment (hmhealey) [inline webapp/channels/src/components/suggestion/suggestion_list.tsx:232]:
```suggestion
                        b: (chunks: string) => <b>{chunks}</b>,
```

**Review by hmhealey — APPROVED** [review]
Thanks for the help with getting rid of this component! There's a small style error, but I'll fix it from here to save some back and forth

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**